### PR TITLE
[fr] New-words-week-49

### PR DESCRIPTION
**Pull request by GillouLT**
To address: https://github.com/languagetooler-gmbh/languagetool-premium/issues/5240
To address and close: https://github.com/languagetooler-gmbh/recurring-issues/issues/32
Linked to premium: https://github.com/languagetooler-gmbh/languagetool-premium-modules/pull/1208

I'm not fully on point with dictionary additions, I might have done some unnecessary or wrong things!

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/ignore.txt [hunk 1]
```diff
@@ -450,4 +450,10 @@ taboire
 tabarnane
 tigidou
 cheum
-cheums
\ No newline at end of file
+cheums
+ROC
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/ignore.txt:454]: Don't we have another file for acronyms?

Comment (LucieSteib) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/ignore.txt:454]: precision: depends if you want to use them as words (like, idk, "RATP", "CAF", which need a gender and number usually). If not ignore.txt is a good place.

**Review by LucieSteib — CHANGES_REQUESTED** [review]
Hi there! 
Here's a few questions :) 
(And **To address and close**: https://github.com/languagetooler-gmbh/languagetool-premium/issues/5240  might not be a good idea, this ongoing issue is suppose to stay open)

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt [hunk 2]
```diff
@@ -4044,4 +4044,15 @@ sandbox;sandbox;N f s
...
 plantogènes;plantogène;J e p
-Baudin;Baudin;Z e sp
\ No newline at end of file
+Baudin;Baudin;Z e sp
+Opera;Opera;Z e sp
+localhost;localhost;N m s
+link;link;N m s
+links;link;N m p
+Syncro;Syncro;Z e sp
+BETA;BETA;Z e sp
```
[the remark this answers is not in the document]
Comment (GillouLT) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt]: True, I got a bit confused by the all caps, but it was probably not referring to anything particular, changed !

**Comment by GillouLT** [comment]
Changes applied, merging :)

**Pull request merged**